test/configSpec.js

```diff
@@ -189,4 +189,23 @@ describe('loadConfig', () => {
       expect(loadConfig).to.throw(/Cannot support precision > 15/)
     })
   })
+
+  describe('config.recommendedConnectors', () => {
+    it('defaults to undefined', () => {
+      const _config = loadConfig()
+      expect(_config.recommendedConnectors).to.equal(undefined)
+    })
+
+    it('is undefined when recommendedConnectors is "*"', () => {
```

Comment:
Got it. What happens if you configure `RECOMMENDED_CONNECTORS` with an account that doesn't have the `connector` field set?